spec/parsers/wav_parser_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+
+describe FormatParser::WAVParser do
+  # Fixtures prefixed with c_ are considered canonical
+  # while fixtures prefixed with d_ deviate from the standard.
+  Dir.glob(fixtures_dir + '/WAV/c_*.*').each do |wav_path|
```

Comment:
Are these files creative commons / public domain licensed / do we have permission to use them? If not let's swap 'em out for ones we're able to use. I can spin up a few files for you if you'd like, just let me know. :)

Reply:
@bermannoah Sorry for this licensing mess.

Here are the websites of origin and licenses of the files:
* c_11k16bitpcm.wav, c_8kmp316.wav are from [Wikipedia WAV article](https://en.wikipedia.org/wiki/WAV#Comparison_of_coding_schemes) and their license don't seem to be explicitly mentioned anywhere. They've been since 22 June 2007 on Wikipedia. 
* c_39064__alienbomb__atmo-truck.wav is from [freesound](https://freesound.org/people/alienbomb/sounds/39064/) and it has CC0 license.
* c_M1F1-Alaw-AFsp.wav, d_6_Channel_ID.wav are from a [McGill Engineering site](http://www-mmsp.ece.mcgill.ca/Documents/AudioFormats/WAVE/Samples.html) and their license is not clear.

Let me know if you consider that I should remove the files except the one licensed under CC0. And if you want to provide me some files that's also welcome. Otherwise I will make some time tomorrow night for searching/creating some free to use files. I guess [this](https://en.wikipedia.org/wiki/List_of_free_content_licenses#For_other_content_types,_including_audio_and_video) is a good list of acceptable licenses.

Reply:
No worries. Thanks much! We do need the files to be able to be licensed/released under the MIT license. A lot of our existing spec files are from other open source (MIT licensed) repos, so you could see what else is out there -- or I can make a few lil files later today, just let me know. :)

Reply:
I really hoped that CC license was compatible with this project. But in this case I will create some files later today. Probably it's quicker than finding MIT licensed WAV files.

Reply:
@bermannoah Would that help if we made a list of fixtures we use and note where they come from and their licenses? I am almost certain CC with attribution is fine, and in case of a claim we will be able to quickly remove those assets.

Reply:
Yes! I think so. I added a few to the readme a while back - but I wasn't sure if we could use CC under MIT. I guess if it's CC0 then we're good, or CC with attribution. (But then does our MIT license mean that future users can use them without attribution?)

Reply:
I'd say let's add a list of all the fixtures including their provenance and if there is a cease-and-desist against any of them we pull them on request. The gems we deliver do not have these files anyway since they get stripped during the gem build.

Reply:
@julik Sounds good. And so then for @duplamatyi -- if he documents the sources/licenses of the existing files then I guess we're good? :)